tokio/src/util/blocking_check.rs

```diff
@@ -0,0 +1,29 @@
+#[cfg(unix)]
+use std::os::fd::AsFd;
+
+#[cfg(unix)]
+#[allow(unused_variables)]
+#[track_caller]
+pub(crate) fn check_socket_for_blocking<S: AsFd>(s: &S) -> crate::io::Result<()> {
+    #[cfg(not(tokio_allow_from_blocking_fd))]
+    {
+        let sock = socket2::SockRef::from(s);
+
+        debug_assert!(
+            sock.nonblocking()?,
+            "Registering a blocking socket with the tokio runtime is unsupported. \
+            If you wish to do anyways, please add `--cfg tokio_allow_from_blocking_fd` to your \
+            RUSTFLAGS. See 7172 for more details."
```

Comment:
```suggestion
            RUSTFLAGS. See #7172 for more details (github.com/tokio-rs/tokio/issues/7172)."
```

How about including the link for those who don't realize 7172 references an issue?

Reply:
Many modern terminals will make URLs clickable as well.

Reply:
I don't think the `https://` is needed. It should end up getting parsed without it and default to https.